snyk-operator-certified/Makefile

```diff
@@ -0,0 +1,92 @@
+# Current Operator version
+VERSION ?= 1.40.3
```

Comment:
Is this being passed as env var anywhere?

Reply:
Yes - you set it in your terminal before running the Makefile commands 🙏